src/lib/lwip/safe_udp.h

```diff
@@ -7,7 +7,7 @@
 #error LWIP_TCPIP_CORE_LOCKING MUST be enabled to use udp_sendto safely!
 #endif
 
-inline err_t safe_udp_sendto(struct udp_pcb *pcb, struct pbuf *p,
+static inline err_t safe_udp_sendto(struct udp_pcb *pcb, struct pbuf *p,
```

Comment:
The inline hint wasn't being picked up by the new compiler, resulting in linker errors. The addition of `static` makes more explicit that it's ok to duplicate the code inline in multiple translation units if necessary.